From 9c9d400049b6823145ed23019843ad771674c0a3 Mon Sep 17 00:00:00 2001
From: Janusz Lisiecki <39967756+JanuszL@users.noreply.github.com>
Date: Mon, 28 Jun 2021 14:22:45 +0200
Subject: [PATCH] Fix DALI installation for python 3.9 version (#3092)

- python_requires has the upper python version set to 3.9
  so any 3.9.x falls beyond that check. Rework the check
  to any 3.9.x based version.

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>
---
 dali/python/setup.py.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dali/python/setup.py.in b/dali/python/setup.py.in
index 09b90f96d9..295fd02204 100644
--- a/dali/python/setup.py.in
+++ b/dali/python/setup.py.in
@@ -1,4 +1,4 @@
-# Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2017-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
@@ -23,7 +23,7 @@ setup(name='nvidia-dali@DALI_FLAVOR_MINUS@-cuda@CUDA_VERSION_SHORT_DIGIT_ONLY@',
       packages=find_packages(),
       include_package_data=True,
       zip_safe=False,
-      python_requires='>=3.6, <=3.9',
+      python_requires='>=3.6, <3.10',
       classifiers=[
           'Programming Language :: Python :: 3.6',
           'Programming Language :: Python :: 3.7',
